src/test/java/com/hocs/test/pages/markup/MarkupFull.java

```diff
@@ -22,4 +28,19 @@ public void moveCaseFromMarkupToInitialDraft() {
         clickFinishButton();
     }
 
+    public void markupStageFullFlow() {
+        homepage.getCurrentCase();
+        clickOn(workstacks.allocateToMeButton);
+        clickOn(markUpDecision.policyResponseRadioButton);
+        clickOn(continueButton);
+        clickOn(topics.addTopicButton);
+        topics.enterRealTopic();
+        waitABit(1000);
```

Comment:
Are these absolutely necessary? I've found that some tests still run with the waits commented out so maybe check if they're necessary